src/Configuration/Dotenv.hs

```diff
@@ -8,17 +8,17 @@
 -- Portability :  portable
 --
 -- This module contains common functions to load and read dotenv files.
-
 {-# LANGUAGE RecordWildCards #-}
 
 module Configuration.Dotenv
-    ( -- * Dotenv Load Functions
-      load
-    , loadFile
-    , parseFile
-    , onMissingFile
+      -- * Dotenv Load Functions
+  ( load
```

Comment:
@FranzGB I think the comment should go inside the parenthesis. Could you please double check the haddock documentation?

Reply:
Sure I will take a look at it